packages/eas-cli/src/graphql/client.ts

```diff
@@ -19,6 +23,16 @@ type SessionHeaders = {
 
 export const graphqlClient = createUrqlClient({
   url: getExpoApiBaseUrl() + '/--/graphql',
+  exchanges: [
+    dedupExchange,
+    cacheExchange,
+    retryExchange({
+      maxDelayMs: 4000,
+      retryIf: err =>
+        !!(err && (err.networkError || err.graphQLErrors.some(e => e?.extensions?.isTransient))),
+    }),
+    fetchExchange,
+  ],
```

Comment:
`dedupExchange`, `cacheExchange`, and `fetchExchange` are implicit defaults that we need to explicitly specify now that we are adding exchanges:
<img width="622" alt="Screen Shot 2021-04-09 at 8 11 31 am" src="https://user-images.githubusercontent.com/1220444/114178250-7285e680-990b-11eb-9528-1397614f5194.png">

https://formidable.com/open-source/urql/docs/architecture/

Reply:
The `retryExchange` must be placed prior to the fetchExchange and after the cacheExchange: https://formidable.com/open-source/urql/docs/advanced/retry-operations/